docs/scenarios_authoring/Spel.md

```diff
@@ -75,9 +75,9 @@ also in some context `Collection` can be met (it's Java API for handling lists,
 See [Handling data/time](#handling-datetime) for detailed description of how to deal with date and time in Nussknacker.
 
 
-# SpEL syntax
```

Comment:
Only title of the page can be at level one. The result is that document structure is the navigation panel is messed up and incomplete.